content/influxdb/v2.1/get-started.md

```diff
@@ -19,6 +19,16 @@ After you've [installed InfluxDB OSS](/influxdb/v2.1/install/), you're ready to
 - [Visualize data](#visualize-data)
 - [Monitor and alert](#monitor-and-alert)
 
+**NOTE:** You **must** start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.1/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.1/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance. 
+
+To start Influx do the following:
+  1. Open a terminal.
+  2. Type influxd in the command line.
+
+```bash
+influxd
```

Comment:
to be consistent in our code examples, we typically use `sh` in lieu of `bash`
(unless there's a technical distinction to be made (eg, you must use `bash` bc `sh` doesn't support that functionality).

Reply:
thanks Kelly